dashboard/test/controllers/transfers_controller_test.rb

```diff
@@ -3,285 +3,228 @@
 
     @word_section = create(:section, user: @teacher, login_type: 'word')
-    @word_user_1 = create(:follower, section: @word_section).student_user
+    @word_student = create(:follower, section: @word_section).student_user
 
     @picture_section = create(:section, user: @teacher, login_type: 'picture')
-    @picture_user_1 = create(:follower, section: @picture_section).student_user
-  end
+    @picture_student = create(:follower, section: @picture_section).student_user
 
-  test "returns an error when student ids are not provided" do
-    post :create, params: {
-      new_section_code: @picture_section.code,
+    @params = {
+      student_ids: @word_student.id.to_s,
       current_section_code: @word_section.code,
-      stay_enrolled_in_current_section: false
+      new_section_code: @picture_section.code
     }
+
+    @other_teacher = create :teacher
+    @other_teacher_section = create :section, user: @other_teacher, login_type: 'word'
+  end
+
+  test "returns an error when student ids are not provided" do
+    @params.delete(:student_ids)
+    post :create, params: @params
     assert_response 400
     assert_equal "Please provide student_ids.", json_response["error"]
   end
 
-  test "returns an error when the section id is invalid" do
-    code = "QWERTY"
-    post :create, params: {
-      new_section_code: code,
-      student_ids: @word_user_1.id.to_s,
-      current_section_code: @word_section.code,
-      stay_enrolled_in_current_section: false
-    }
-
+  test "returns an error when new_section_code is invalid" do
+    @params[:new_section_code] = NONEXISTENT_SECTION_CODE
+    post :create, params: @params
     assert_response 404
-    assert_equal "Sorry, but section #{code} does not exist. Please enter a different section code.", json_response["error"]
+    assert_equal(
+      "Sorry, but section #{NONEXISTENT_SECTION_CODE} does not exist. Please "\
+        "enter a different section code.",
+      json_response["error"]
+    )
   end
 
-  test "returns an error when the current_section_code is not provided" do
-    post :create, params: {
-      new_section_code: @picture_section.code,
-      student_ids: @word_user_1.id.to_s,
-      stay_enrolled_in_current_section: false
-    }
+  test "returns an error when current_section_code is not provided" do
+    @params.delete(:current_section_code)
+    post :create, params: @params
     assert_response 400
     assert_equal "Please provide current_section_code.", json_response["error"]
   end
 
   test "returns an error when stay_enrolled_in_current_section is not provided and a student is being transferred to another teacher" do
-    new_teacher = create(:teacher)
-    new_section = create(:section, user: new_teacher, login_type: 'word')
-
-    post :create, params: {
-      new_section_code: new_section.code,
-      student_ids: @word_user_1.id.to_s,
-      current_section_code: @word_section.code
-    }
+    @params[:new_section_code] = @other_teacher_section.code
```

Comment:
The test on line 104 seems to be testing the same thing. Can we get rid of this one?